simulation-runner/config/nodes/pe201523-catalogzero.json

```diff
@@ -0,0 +1,6 @@
+{
+    "certname": "w3h244u9nwii0y5.delivery.puppetlabs.net",
+    "simulation_class": "com.puppetlabs.gatling.node_simulations.PE201523CatalogZero",
```

Comment:
Don't worry, already merged. It's here: https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/PE201523CatalogZero.scala